samples/demo-config.js

```diff
@@ -18,7 +18,7 @@ module.exports = {
         path: '/maps/'
     },
     'Highcharts Stock': {
-        categories: ['General', 'Chart types', 'Various features', 'Flags and Technical indicators', 'Designs'],
+        categories: ['General', 'Chart types', 'Various features', 'Flags and Technical indicators', 'Designs', 'Crypto'],
```

Comment:
We dediced not to add a Crypto category, because the demo page categories are based on features, not content. Some of the crypto demos will be featured in a blog article. Not sure where it stands with this particular demo.